store/sqlstore/group_store.go

```diff
@@ -312,6 +312,40 @@ func (s *SqlGroupStore) GetMemberCount(groupID string) (int64, *model.AppError)
 	return count, nil
 }
 
+func (s *SqlGroupStore) GetMemberUsersNotInChannel(groupID string, channelID string) ([]*model.User, *model.AppError) {
```

Comment:
Is this query too specific and should maybe be moved into `GetMemberUsers` with some options struct? to achieve the same behaviour?

Reply:
I don't have any opinion on wether or not it should be moved but I would say it makes more sense if it was called `GetGroupMemberUseresNotInChannel`.

Reply:
The reason I called it `GetMemberUsersNotInChannel ` is to match the style of the `GetMemberUsers` function which gets all of a groups users